WordPress/src/main/java/org/wordpress/android/util/image/ImageManager.kt

```diff
@@ -47,6 +50,14 @@ class ImageManager @Inject constructor(val placeholderManager: ImagePlaceholderM
                 .into(imageView)
     }
 
+    fun load(viewTarget: ViewTarget<TextView, Drawable>, imageType: ImageType, imgUrl: String) {
```

Comment:
Just curious, is it at all possible to avoid this many different `load` methods in `ImageManager`? Can we use a builder pattern or named parameters with default values to avoid it? This, of course, doesn't need to be part of this PR, but just a general concern.

Reply:
I've consider it before starting this project, however I felt it'd be nice to have limited options how to load images, so we can easily be consistent across the app. I also didn't want to have any Glide classes in the interface, but it could be solved by wrapping GlideRequest, I guess. However, I don't have a strong opinion on this. Wdyt?

Reply:
I think some kind of refactor would be great for this. I have to look into it more to figure out what the best way might be, but I think we simply have too many load functions right now.

As I said, this is not related to this PR, but something to consider before we finalize the Glide project.

Reply:
I'd appreciate any help with it. We might allocate 30 minutes and resolve it during a call. Wdys?

Reply:
Sure thing. Do you want to open an issue for this and assign it to both of us, so we can check it out once all Glide PRs are merged in?